src/lib/process_crawl.js

```diff
@@ -1,27 +1,197 @@
 var rc_util = require('rippled-network-crawler/src/lib/utility.js');
+var crawler = require('rippled-network-crawler/src/lib/crawler.js');
```

Comment:
Cool, I see now that a lot of this was already just copied from the crawler's utility.js